### PR TITLE
[ASA-366][JR] Adding state pension's payment frequency and start date of weekly payments

### PR DESCRIPTION
**Pull request by johnrevill**

### CODE REVIEW

File: app/uk/gov/hmrc/taxhistory/model/nps/IabdType.scala [hunk 1]
```diff
@@ -17,6 +17,9 @@
 package uk.gov.hmrc.taxhistory.model.nps
 
 
+import org.joda.time.LocalDate
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/taxhistory/model/nps/IabdType.scala]: can we use java 8 time please ?

File: app/uk/gov/hmrc/taxhistory/model/nps/IabdType.scala [hunk 2]
```diff
@@ -123,9 +126,31 @@ case class Iabd(nino: String,
                 `type`: IabdType,
                 grossAmount : Option[BigDecimal] = None,
                 typeDescription : Option[String] = None,
-                source: Option[Int] = None) {
-
-  def toStatePension = StatePension(grossAmount = grossAmount.getOrElse(0.0), typeDescription.getOrElse(""))
+                source: Option[Int] = None,
+                paymentFrequency: Option[Int],
+                startDate: Option[String]) {
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/taxhistory/model/nps/IabdType.scala]: can we assign `None` by default for these new fields ?

Comment (johnrevill) [reply app/uk/gov/hmrc/taxhistory/model/nps/IabdType.scala]: Yep, done.

**Comment by sureshhmrc** [comment]
LGTM

**Comment by sureshhmrc** [comment]
sorry,, i've just realised that i don't have the merge perms on this repo

**Comment by johnrevill** [comment]
@sureshhmrc Should be good now, I've just added you to the PFA team

**Pull request merged**